docs/modules/game-object-creation/overview.md

```diff
@@ -0,0 +1,27 @@
+# GameObject Creation Feature Module
+
+This feature module contains an default implementation of spawning GameObjects for your SpatialOS entities and offers the ability to customize that process.
+
+## Installation
+
+### 1. Add the package
+
+To add this feature module to your project, add the following line to the `dependencies` section in your `workers/my-unity-project/Packages/manifest.json`:
+
+```json
+    "com.improbable.gdk.gameobjectcreation": "file:<path-to-the-gdk>/workers/unity/Packages/com.improbable.gdk.gameobjectcreation",
+```
+
+> **Note:** `<path-to-the-gdk>` is the relative path between your `manifest.json` and the `gdk-for-unity` project.
```

Comment:
this took me a while to figure out what exactly is meant here. Should we link to the gdk repo to make it absolutely clear that this is not some random project using the gdk, but the actual gdk repo.

I would also assume that many peoople are not aware that there is a `manifest.json` in the Packages folder as you would usually do it via the Unity Editor.
On that note, why not add instructions on how to add it via Unity Editor instead of the manifest.json? :D

Reply:
Can you add it to the manifest using the Unity Editor? That's news to me 😂